lib/lua/lua_init.lua

```diff
@@ -91,6 +91,17 @@ commands.register_key = function(key)
     end
 end
 
+-- unregisters a keybinding
+commands.unregister_key = function(key)
```

Comment:
Using `commands.key` for this requires specifying an action: `unregister_key(key({ mod }, "x", ?))`.
Having a way that does not require specifying one would be more ergonomic. Maybe the check for the type of action should be moved to `commands.register_key`.